src/components/CreateRules/UploadRulesSummary.vue

```diff
@@ -1,54 +1,48 @@
 <template>
-  <v-card flat>
-    <!-- Summary Section -->
-    <div id="upload-rules-summary">
-      <!-- Summary Header -->
-      <div class="upload-rules-summary-header" >
-        <v-icon color="appDkBlue">mdi-format-list-text</v-icon>
-        <label class="upload-rules-title pl-2"><strong>Rules</strong></label>
-      </div>
-      <!-- Summary error message -->
-      <div
-        v-if="!getCreateRulesStep.valid"
-        class="upload-rules-error-message invalid-section"
-      >
-      <span>
-        <v-icon color="error">mdi-information-outline</v-icon>
-        &nbsp;
-        <span class="error-text">This step is unfinished.</span>
-        &nbsp;
-        <router-link
-          :to="{ path: `/${RouteNames.CREATE_RULES}` }"
-        >Return to this step to finish it</router-link>
-      </span>
+  <v-card id="upload-rules-summary-card" flat class="rounded-0">
+    <div class="upload-rules-summary-header review-header">
+      <v-icon color="appDkBlue">mdi-format-list-text</v-icon>
+      <label class="upload-rules-title pl-2"><strong>Rules</strong></label>
+    </div>
+
+    <div :class="{ 'invalid-section': !getCreateRulesStep.valid }">
+      <div v-if="!getCreateRulesStep.valid" class="upload-rules-error-message">
+        <span>
+          <v-icon color="error">mdi-information-outline</v-icon>
+          <span class="error-text">This step is unfinished.</span>
+          <router-link
+            :to="{ path: `/${RouteNames.CREATE_RULES}` }"
+          >Return to this step to finish it</router-link>
+        </span>
       </div>
     </div>
+
+    <div v-if="getCreateRulesStep.valid" class="upload-rules-success-message">
+      <v-row no-gutters>
+        <v-col md="1">
+          <v-icon class="upload-success-chk">mdi-check</v-icon>
+        </v-col>
+        <v-col md="11" id="file-name-col">
+          <span id="file-name">{{getCreateRulesStep.rulesDoc.name}}</span>
+        </v-col>
+      </v-row>
+    </div>
   </v-card>
 </template>
 
 <script lang="ts">
 // Libraries
-import { Component, Mixins, Prop } from 'vue-property-decorator'
-import { Action, Getter } from 'vuex-class'
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
 
 // Enums
 import { RouteNames } from '@/enums'
 import { CreateRulesIF } from '@/interfaces'
 
-// Mixins
-import { DocumentMixin } from '@/mixins'
-
-// Components
-import FileUploadPreview from '@/components/common/FileUploadPreview.vue'
-
 @Component({
-  components: {
-    FileUploadPreview
-  }
+  components: {}
```

Comment:
I'm not sure if this is required if empty.

Reply:
will test without.

Reply:
updated to just use `@Component()`

Reply:
LGTM